server/schemas.js

```diff
@@ -35,12 +35,14 @@ const Authentication = mongoose.model('Authentication', authenticationSchema);
 
 const eventSchema = new mongoose.Schema({
     eventID: { type: String, required: true, unique: true},
-    eventName: { type: String, required: true},
-    location: { type: String },
-    collaborators: [{ type: String }],
-    viewers: [{ type: String }],
+    title: { type: String, required: true},
+    location: [ { address: String } ],
+    collaborators: [{ username: String, name: String }],
+    viewers: [{ username: String, name: String }],
     date: { type: Date},
-    past: { type: Boolean }
+    past: { type: Boolean },
+    description: { type: String },
+    color: { type: String, default: '#FFFFFF' }
```

Comment:
If it isn't too big of a deal, could you please update the default to be "#8be4d2"? (It's the main app color)

Reply:
done!